fix: Doc about installing ImagePuller (#2835)
codenvy/che-docs: modules/administration-guide/pages/installing-image-puller-on-openshift-using-cli.adoc

You can install the {image-puller-name} on OpenShift by using OpenShift `oc` management tool.

[IMPORTANT]
====

> If the ImagePuller is installed with the `oc` CLI, it cannot be configured via the `CheCluster` Custom Resource.

Reviewer: ```suggestion
If the ImagePuller was installed with the `oc` CLI, you can not configure it via the `CheCluster` Custom Resource.
```
